.github/workflows/labeler.yaml

```diff
@@ -1,10 +1,15 @@
-name: Issue Labeler
+name: Issue and PRs Labeler
 on:
   issues:
     types: [opened, edited]
   pull_request:
     types:
+      - opened
+      - reopened
+      - synchronize
       - ready_for_review
+    paths:
+      - 'docs/**'
```

Comment:
yes, previously it was there but the regex didn't work see this [log ](https://github.com/andreaangiolillo/mongocli-test/actions/runs/8910528589/job/24469977619#step:4:159) of a PR changing the doc file

Reply:
Adding more context why the regex in `labeler.yaml` was `docs\\/*.txt`

I was following their [documenation](https://github.com/srvaroa/labeler?tab=readme-ov-file#files-affected-prs-only-)

> Files affected (PRs only)
> This condition is satisfied when any of the PR files matches on the given regexs.
> 
> files: 
> - "cmd\\/.*_tests.go"
> - ".*\\/subfolder\\/.*\\.md"
> NOTICE the double backslash (\\) in the example above. This GitHub Action is coded in Go (Golang), which means you need to pay special attention to regular expressions (Regex). Special characters need to be escaped with double backslashes. This is because the backslash in Go strings is an escape character and therefore must be escaped itself to appear as a literal in the regex.

Reply:
nvm. I managed to do it in the latest commit